java/ql/test/stubs/spring-context-5.3.2/org/springframework/stereotype/Controller.java

```diff
@@ -0,0 +1,14 @@
+package org.springframework.stereotype;
```

Comment:
Spring stubs: can you add to / use the existing https://github.com/github/codeql/tree/main/java/ql/test/stubs/springframework-5.2.3 rather than add new subdirectories for these Spring subpackages?

Reply:
I use `SpringBoot`. It can be modified, but I don't think it will affect the evaluation.

Reply:
Indeed, this stuff is purely for getting the tests to run, in which context I don't think Spring Boot is relevant?

Reply:
I will make a new stub under the `springframework-5.2.3` folder, tentatively before next Monday.

Reply:
I have changed `springframework-5.2.3` stub and tested